plugins/org/README-alpha.md

```diff
@@ -0,0 +1,462 @@
+> If you are not using the new frontend system, please go [here](./README.md).
+
+This is a plugin that extends the Catalog entity page with some users and groups overview cards:
+
+- Group Profile Entity Card
+- Member List Entity Card
+- Ownership Entity Card
+- User Profile Entity Card
+
+Here is a Catalog group page showing the group profile, members, and ownership cards:
+
+![Group Page example](./docs/group-page-example.png)
+
+And below is an example of how a user page looks with the user profile and ownership cards:
+
+![Group Page example](./docs/user-profile-example.png)
+
+## Table of Content
+
+- [Installation](#installation)
+- [Packages](#packages)
+- [Routes](#routes)
+- [Extensions](#extensions)
+  - [Entity Group Profile Card](#entity-group-profile-card)
+  - [Entity Group Profile Card](#entity-members-list-card)
+  - [Entity Group Profile Card](#entity-members-list-card)
+  - [Entity Group Profile Card](#entity-user-profile-card)
+  - [My Groups Sidebar Item](#my-groups-sidebar-item)
+
+## Installation
+
+1. Install the `org` plugin in you Backstage app:
+
+   ```bash
+   # From your Backstage root directory
+   yarn --cwd packages/app add @backstage/plugin-org
+   ```
+
+2. Enable which entity cards and tabs you would like to see on the catalog entity page:
+
+   ```yaml
+   # app-config.yaml
+   app:
+     experimental:
+       # Auto discovering all plugins extensions
+       packages: all
+     extensions:
+       # Enabling the org plugin cards
+       - entity-card:org/group-profile
+       - entity-card:org/members-list
+       - entity-card:org/ownership
+       - entity-card:org/user-profile
+   ```
+
+3. Then start the app, navigate to an entity's page and see the cards and contents in there.
+
+## Packages
+
+The `org` plugin can be automatically discovered, and it is also possible to enable it only in certain [environments](https://backstage.io/docs/conf/writing/#configuration-files). See [this](https://backstage.io/docs/frontend-system/architecture/app/#feature-discovery) packages documentation for more details.
+
+## Routes
+
+The `org` plugin exposes a external route that can be used to configure route bindings.
+
+| Key            | Type           | Description                        |
+| -------------- | -------------- | ---------------------------------- |
+| `catalogIndex` | External route | A route ref to Catalog Index page. |
+
+As an example, here is an association between the external catalog index page and a regular route from another plugin:
+
+```yaml
+# app-config.yaml
+app:
+  routes:
+    bindings:
+      # example binding org and catalog index pages
+      org.catalogIndex: catalog.catalogIndex
+```
+
+Route binding is also possible through code. For more information, see [this](https://backstage.io/docs/frontend-system/architecture/routes#binding-external-route-references) documentation.
+
+## Extensions
+
+### Entity Cards
```

Comment:
we go all the way to five levels of headings which is a lot - maybe it's worth skipping this grouping and just putting all extensions in a single list? 🤔 not sure

Reply:
Yes, no strong opinion. I grouped the cards to add a note that the card order is defined by the setting, but I think I can repeat the note for each card. Do you have any preferences 🙂?

Reply:
Not a strong one. Just that more than like three or mmmaybe four levels quickly gets to a point where it's not possible to visually tell the levels apart anymore

Reply:
Adjusted 🙌🏻